### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by DmytroBondariev**

### CODE REVIEW

**Comment by Dmytry95** [comment]
image

image

image

image


image

File: .flake8 [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [flake8]
 inline-quotes = "
 ignore = E203, E266, W503, N807, N818, F401
-max-line-length = 79
+max-line-length = 100
```
Comment (ArsenPidhoretskyi) [inline .flake8:4]: No need to change this line

File: taxi/forms.py [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+#     #         )
+#     #     return license_number
+#
+#     class Meta(UserCreationForm.Meta):
+#         model = Driver
+#         fields = UserCreationForm.Meta.fields + (
+#             "first_name",
+#             "last_name",
+#             "license_number",
+#         )
```
Comment (ArsenPidhoretskyi) [inline taxi/forms.py:45]: No need to store commented code

Comment (ArsenPidhoretskyi) [reply taxi/forms.py:45]: Fix in all places

**Review by ArsenPidhoretskyi — CHANGES_REQUESTED** [review]
html files must end with empty line

**Review by ArsenPidhoretskyi — APPROVED** [review]
Well done!

**Pull request closed without merging**